foundation/foundation-mda/src/test/resources/specifications/extend-metamodel.feature

```diff
@@ -0,0 +1,15 @@
+@extend-metamodel
+Feature: Extend foundation-mda so that you can modify existing metamodel schemas
+
+  Scenario Outline: Foundation-mda metamodel elements are extendable
+    Given a class extending the "<type>" metamodel with new simple and complex fields
+    When the "<profile>" profile is generated
```

Comment:
A: This feels like it has the potential to clash and the profiles we've selected feel a little arbitrary.  It's extra coverage so I'm OK leaving it, but I'd also be OK with just making this _"When the extended models are read in"_ or something to that effect.

Reply:
The reason I added it was cause the error itself occurs when the generators are run, so felt like the tests we insufficient if we didn't include them.